docs/autogenerated-menu/Our Practices/Engineering/Procedures/Document Demonstrate Duplicate.md

```diff
@@ -0,0 +1,41 @@
+
+After you train or handover a few times this way, you’ll have worked out the kinks for that job and it’s pretty smooth sailing from there. 
+_Think about it this way, if you vanished tomorrow, could a new person get the results you get if they only followed your guide? That’s the level of clarity to shoot for._
+
+## Training Notes
+
+A helpful way to look at this training style is: If they get it wrong or get confused then we got it wrong or made it confusing. If we have to explain what a step means then the step is too complicated. Or, more likely, we tried to put multiple steps into one.
+
+If they only appear to “get it” after a longish explanation or multiple demonstrations then, again, we’ve got some work to do. Anyone that ignore this run into chronic training problems. And, word to the wise, you can probably force an inferior guide to work, but this turns into a nightmare when somebody else takes over your training for you.   
+
+There is a difference between competence and performance. In other words, they can know exactly what to do and not be that good at it yet. If that’s the case, then your instructions are fine and they just need practice. You don’t need to change anything, they just need more reps.
+
+Focus on your trainee’s ability to follow directions more than whether they get the right result. This is super important because if you train your trainees to follow directions then… they will follow directions. And, if they follow directions and get the wrong result… then you know it’s the directions. That’s good. You have a lot more control over that.
+
+Every time they do a step successfully–let them know they did it right. And if they respond to praise, praise them! And if they goof, that’s OK too. That’s what training is for. Don’t take over for them when they mess up–simply pause, refocus, and let them try it again. Fast feedback cycles to get people to learn faster.
+
+If they follow your directions exactly and get the wrong result–still praise them for following the directions. Praise them, then make the corrections to your guide on the spot.
+
+Avoid punishment or penalties of any type for doing stuff wrong during training. As a rule of thumb–reward the good stuff you want them to do more of and they'll do more of it. Learning a new skill is punishing enough, we don't need to add to it.
+
+It’s hard to fix multiple things when you've never done something before. Give feedback one step at a time. Give one piece of feedback at a time. Practice until they get it right. Then, move to the next step.   Whenever there is a major dip from normal performance, retrain the team. They stopped doing an important step in the process (often because they didn’t know it was important). Once you figure out the step, reward people for following it going forward.
```

Comment:
```suggestion
It’s hard to fix multiple things when you've never done something before. Give feedback one step at a time. Give one piece of feedback at a time. Practice until they get it right. Then, move to the next step. Whenever there is a major dip from normal performance, retrain the team. They stopped doing an important step in the process (often because they didn’t know it was important). Once you figure out the step, reward people for following it going forward.
```